test/linter/test-consistency.js

```diff
@@ -97,7 +122,7 @@ const chalk = require('chalk')
     // Test whether sub-features are supported when basic support is not implemented
     // For all unsupported browsers (basic support == false), sub-features should be set to false
     const supportUnknownInParent = this.extractSupportUnknownBrowsers(data.__compat);
-    var inconsistentSubfeaturesByBrowser = {};
+    inconsistentSubfeaturesByBrowser = {};
```

Comment:
Here.

Reply:
This&nbsp;is&nbsp;a&nbsp;bug that&nbsp;prevents the&nbsp;**JSDoc**&nbsp;type from&nbsp;applying to&nbsp;this&nbsp;variable in&nbsp;the&nbsp;whole&nbsp;function.

Reply:
Not saying that these aren’t problems, just that they should be cherry-picked.  🙂

Reply:
This&nbsp;fix is&nbsp;in&nbsp;scope.

Reply:
Personally, I feel that it's not really in the scope as it talks about just fixing the JSDoc types, not the variable assignments.  But eh, in the end, all that matters is that it's fixed!